app/main/views/frameworks.py

```diff
@@ -207,3 +219,20 @@ def _framework_updates_page(error_message=None):
         error_message=error_message,
         **template_data
     ), status_code
+
+
+def _get_declaration_status():
```

Comment:
I put the method there to start with but I didn't like the feel of making api calls from helpers so moved it back here... I know it's a bit awkward here, but at least it has the api client, etc. to hand. 
What do you think?  It should move anyway?